OffsetLib/gxy.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+
+import ktl
+
+from wftel import wftel
+import KeckLogger
+
+parser = argparse.ArgumentParser(description="Move the telescope guider coordinates",
+                         usage="gxy.py [-n] tvxoff tvyoff")
+
+parser.add_argument("-n", "--nomove", action='store_true',
+                         help="don't send moves")
+
+parser.add_argument("tvxoff", type = float, help="guider x offset")
+
+parser.add_argument("tvyoff", type=float, help="guider y offset")
+
+args = parser.parse_args()
+
+def gxy(n, x, y):
```

Comment:
`n` should be a more clearly named thing since we're no longer bound by the old command line formatting from the original script.  Perhaps renaming `n` to `nomoves`: `def gxy(x, y, nomoves=False):`